db/migrations/20240202124553_AddRepoConfigLabel.up.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+
+
+
+ALTER TABLE repository_configurations
+    ADD COLUMN IF NOT EXISTS label VARCHAR (255)   DEFAULT FALSE ;
+
+COMMIT;
```

Comment:
is default  "false" the intention here or should this be null/blank? Also, if I create a new custom repository, the default is an empty string.

Reply:
good catch, this should be empty string.

Reply:
i'll also set it not null